source/gems/dry-initializer/custom-plugins.html.md

```diff
@@ -91,17 +91,19 @@ require 'dry-initializer'
 
 module Dry::Initializer::Coercion
   class Builder
```

Comment:
I believe `Builder` should inherit from `Dry::Initializer::Plugins::Base`, as the other parts of the docs say.